### PR TITLE
Add new restricted view alert to docs

### PR DESCRIPTION
**Pull request by blackbaud-johnly**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/stache?_sp.spa=pr-restricted-view-alert](https://host.nxt.blackbaud.com/stache?_sp.spa=pr-restricted-view-alert)

**Comment by Blackbaud-LindseyRix** [comment]
There are a few places in these docs that we use the restricted view, should we also add this alert to that area?

File: src/app/components/restricted-view/index.html [hunk 1]
```diff
@@ -32,6 +33,11 @@ <h3>
         <sky-code>*skyRestrictedView</sky-code> — The <sky-code>*skyRestrictedView</sky-code> directive is placed as an attribute on a restricted element. It hides any content within that element from users who are not logged in with BBID.
       </p>
     </li>
+    <li>
+      <p>
+        <sky-code>sky-restricted-content-alert</sky-code> — The <sky-code>sky-restricted-content-alert</sky-code> component displays an <a routerLink="/components/alerts">alert</a> that instructs users to log in to see information that is available only to Blackbaud employees. The alert appears in place of hidden content only when users who are not logged in with BBID previously viewed the content while logged in with BBID.
```
Comment (Blackbaud-TrevorBurch) [inline src/app/components/restricted-view/index.html:38]: Technically the user has to be a Blackbaud employee who logged in with BBID and now isn't for this to show up. It does not show up if a user previously logged in with a non-employee BBID.

**Review by Blackbaud-TrevorBurch — CHANGES_REQUESTED** [review]
@blackbaud-johnly - one question from my point of view.

**Comment by blackbaud-johnly** [comment]
> There are a few places in these docs that we use the restricted view, should we also add this alert to that area?

@Blackbaud-LindseyRix, I added the alert to the search docs, which is the only other place I saw where we had restricted view in place. Feel free to update with any others that you know of. ... Also, can you please peer review this PR? Thanks in advance.

File: src/app/components/restricted-view/index.html (hunk 1 above)
Comment (blackbaud-johnly) [reply src/app/components/restricted-view/index.html]: Updated. Thanks!

**Review by Blackbaud-TrevorBurch — APPROVED** [review]

**Comment by blackbaud-ado** [comment]
Preview build at [https://host.nxt.blackbaud.com/stache?_sp.spa=pr-restricted-view-alert](https://host.nxt.blackbaud.com/stache?_sp.spa=pr-restricted-view-alert)

**Review by blackbaud-ellynhassell — APPROVED** [review]

**Review by Blackbaud-LindseyRix — APPROVED** [review]

**Pull request merged**